store/migration/10001__init_schema.sql

```diff
@@ -840,7 +840,8 @@ CREATE TABLE task (
         )
     ),
     `type` TEXT NOT NULL CHECK (`type` LIKE 'bb.task.%'),
-    payload TEXT NOT NULL DEFAULT ''
+    payload TEXT NOT NULL DEFAULT '',
+    `not_before_ts` BIGINT NOT NULL DEFAULT 0
```

Comment:
I guess we may need to change the index here as well. Because when we are looking for the tasks ready to be executed, we will look for status == xx AND not_before_ts <= current_ts.

And in the future, we will also have a support to list all to be executed tasks,  ORDER BY not_before_ts ASC.

So I would suggest to add an index on this field.

Reply:
Currently, we will create a Timing TaskCheck for each task before executing them.
And when selecting tasks ready to be executed, we will first check whether they have passed all the TaskCheck.
So, statement like `status == xx AND not_before_ts <= current_ts` is actually not used for deciding which task is good to run.

But maybe in some cases, we may adopt this approach. I will create an index for them nonetheless.

Reply:
done, PTAL